src/vscode-cosmosdb.api.d.ts

```diff
@@ -59,7 +59,7 @@ export interface DatabaseTreeItem extends DatabaseAccountTreeItem {
 
 export type CosmosDBResourceType = 'DatabaseAccount' | 'Database';
 
-export type CosmosDBApiType = 'Mongo' | 'SQL' | 'Graph' | 'Table';
+export type CosmosDBApiType = 'Mongo' | 'SQL' | 'Graph' | 'Table' | 'Postgres';
```

Comment:
I would keep this file as-is for now. It defines the API we use between the App Service extension and the Cosmos extension and we want to be careful about any changes because it's a lot harder to change an API after the fact. This won't be needed until you add support for the "Connections" scenario discussed in standup